Tests/Tests/TooltipDispatcherSpec.swift

```diff
@@ -211,21 +211,47 @@ class TooltipDispatcherSpec: QuickSpec {
 
                     context("and permission response display are true") {
 
-                        it("will display the tooltip") {
-                            permissionService.shouldGrantPermission = true
-                            let tooltip = TestHelpers.generateTooltip(id: "test", title: "[Tooltip][ctx] title", isTest: true)
-                            dispatcher.setNeedsDisplay(tooltip: tooltip)
-                            expect(router.displayedTooltips).toEventuallyNot(beEmpty())
+                        context("and isTest are true") {
+
+                            it("will display the tooltip") {
+                                permissionService.shouldGrantPermission = true
+                                let tooltip = TestHelpers.generateTooltip(id: "test", title: "[Tooltip][ctx] title", isTest: true)
+                                dispatcher.setNeedsDisplay(tooltip: tooltip)
+                                expect(router.displayedTooltips).toAfterTimeoutNot(beEmpty())
```

Comment:
When we expect a different state than initial, we can use `toEventually()` which is better than `toAfterTimeout()` because it doesn't passively wait for 1 second. So please use `toEventuallyNot()` here.

Reply:
Thank you for the explanation, I changed `toEventuallyNot ()` to `toEventually ()` in the file the on this commit [afcdff1](https://github.com/rakutentech/ios-inappmessaging/pull/252/commits/8ab06298f635d77318d4cbb90d1131036afcdff1).

Reply:
Thank you 👍